### PR TITLE
Jefjos's first goodie

### PR DESCRIPTION
**Pull request by jefjos**

### CODE REVIEW

**Comment by moollaza** [comment]
@jefjos thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

BTW, can you please fill out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/pull_request_template_spice.md) and edit your initial comment. Once completed, we'll be sure to review your PR and give you some feedback.

Thanks!

**Comment by moollaza** [comment]
@jefjos also, it looks like you somehow duplicated your .pm and .t files -- could you please remove the 2 files you don't need?

File: lib/DDG/Goodie/IsAwesome/jefjos.pm [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/jefjos.pm";
+attribution github => ["https://github.com/jefjos", "jefjos"];
+
+triggers start => "duckduckhack jefjos";
+
+handle remainder => sub {
+
+    # optional - regex guard
```
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/jefjos.pm]: You can remove these commented lines

File: lib/DDG/Goodie/IsAwesome/jefjos.pm [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package DDG::Goodie::IsAwesome::jefjos;
+ABSTRACT: jefjos's first goodie
```
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/jefjos.pm]: this line should be a comment. Please add a `#` to the beginning

File: t/IsAwesome/jefjos.t [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+zci answer_type => "is_awesome_jefjos";
+zci is_cached   => 1;
+
+ddg_goodie_test(
+    [qw(
+        DDG::Goodie::IsAwesome::jefjos
+    )],
+    'duckduckhack jefjos' => test_zci('Jefjos is awesome!'),
+    'duckduckhack GitHubUsername is awesome' => undef,
```
Comment (moollaza) [inline t/IsAwesome/jefjos.t]: Please replace `GitHubUsername` with `jefjos`

File: t/isAwesome/jefjos.t [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env perl
```
Comment (moollaza) [inline t/isAwesome/jefjos.t]: Looks like this file is a duplicate? Please remove.

**Comment by javathunderman** [comment]
Hey @jefjos, are you interested in finishing this up? It would be great to see this go live.

**Comment by MrChrisW** [comment]
@jefjos Closing this for now. Please reopen if you'd like to continue with your First Goodie. Thanks

**Pull request closed without merging**